react-ui/package.json

```diff
@@ -20,5 +20,17 @@
     "eject": "react-scripts eject"
   },
   "proxy": "http://localhost:4000",
-  "devDependencies": {}
+  "devDependencies": {},
+  "browserslist": {
+    "production": [
```

Comment:
Hi Kyle,
Was this added on purpose or just a side effect of running the code? If it was purposeful, just a side note that we will need to check that our production website still works after this PR is merged.

Reply:
that was a side effect of running the code

Reply:
Makes sense. Thank you for removing it!